dist/scss/wds-components/_lists.scss

```diff
@@ -48,5 +48,10 @@
 				padding: $list-item-padding;
 			}
 		}
+
+		.wds-is-link {
```

Comment:
I believe this is one time use-case for global navigation. If we really need it let's put it there instead for add more junk into `wds-list`